lms/djangoapps/courseware/models.py

```diff
@@ -248,3 +254,95 @@ class Meta(object):   # pylint: disable=missing-docstring
 
     field = models.CharField(max_length=255)
     value = models.TextField(default='null')
+
+
+SCORE_CHANGED = Signal(
```

Comment:
Please put comments above this indicating exactly what situations it's emitted in. I'm fine with the notion that it's emitted on scoring events even if the actual score hasn't changed (we can always add a kwarg for that later), but it should be clearly documented.